submodules/wizard/wizard.js

```diff
@@ -1153,6 +1296,160 @@ define(function(require) {
 
 		/* UTILITY FUNCTIONS */
 
+		/**
+		 * Toggles the display of one or more application cards, in an animated way.
+		 * The animation is similar to Isotope, but without having to worry about responsiveness
+		 * thanks to flexbox.
+		 * The animation uses the FLEX technique.
+		 * @param  {Object} args
+		 * @param  {('show'|'hide')} args.action  Show or hide the elements
+		 * @param  {jQuery} args.container  Items container
+		 * @param  {jQuery} args.itemsToToggle  Items to toggle
+		 */
+		wizardToggleAppCard: function(args) {
+			var self = this,
+				animationMillis = 500,
```

Comment:
Curious why this is not abstracted like the other animations time?

Also, half a second seems pretty slow.

Reply:
I forgot that I had already put the times for other animations in the `appFlags`

I moved the times there, and reduced them a little. I left the card animation to 400 millis, because I felt that 350 or less was too quick.